app/gui/view/src/root.rs

```diff
@@ -83,6 +83,11 @@ impl Model {
     fn init_project_view(&self) {
         if self.project_view.get().is_none() {
             let view = self.app.new_view::<crate::project::View>();
+            // This is a quickfix for the bug #180438664.
+            {
+                view.code_editor().toggle();
+                view.code_editor().toggle();
+            }
```

Comment:
This is just a quickfix. I plan to investigate this bug further, but I don't think we should block the entire Welcome Screen until then.